### ISSUE TITLE
Add a "fromPos" argument to PathFollower:Compute

### ISSUE THREAD

**Issue by ghost** [Facepunch/garrysmod-requests#898]
Since it doesn't seem like NextBot players will get updated soon, I'm going to have to resort to using a separate NextBot entity to compute paths for the playerbot entity. However, the problem is that PathFollower:Compute's "from" argument is just a NextBot (It assumes that the "from" position you're specifying is always their position), and unless I want to set the NextBot's pos to the playerbot's position every frame, I need the NextBot to become a separate argument, and have "from" be replaced by a vector.

tl;dr I need
`PathFollower:Compute(NextBot from, Vector to)`
changed to
`PathFollower:Compute(NextBot bot, Vector from, Vector to)`
